src/loader.ts

```diff
@@ -1,136 +1,31 @@
+import {
```

Comment:
You shouldn't need to import anything.  If the `interface.d.ts` is in the scope of the compiler, these types should already be available to you.